EVM and High Level Languages.md

```diff
@@ -0,0 +1,95 @@
+
+## Discrete Parts that make the EVM
+
+As stated before, the EVM is prebuilt with libraries of ready to use functions. Smart contract languages such as Solidity and Vyper are simple, feature-rich languages. Embedded function libraries manipulate data in ways making the EVM quasi-Turing complete. Turing complete system is so named on the basis that it can use a lot of math proving it can solve any algorithm.
+
+### EVM Opcodes and Bytecode
+
+<a href="https://imgur.com/J3lKgEy"><img src="https://i.imgur.com/J3lKgEy.png" title="source: imgur.com" /></a>
+
+Opcodes are a large library of functions preprogrammed into the EVM. A list of all Opcodes available is accessible [here](https://www.ethervm.io/#opcodes). Fewer resources handle these functions because they are small, 4-bit hex identifiers. Every function found in JavaScript or Python exists also in the EVM. Opcodes include functions for data manipulation, I/O, storage, security, and logic. High-level smart contract language compiles into bytecode. This is data passed into the EVM in hex that calls opcode functions and handles user arguments. The image located at the top of the page is a matrix of two digit hex opcodes. 
+
+### Transactions
+
+Smart Contracts are simple to deploy on the EVM blockchain. Every type of transaction uses the same format to execute the EVM. Two main smart contact transactions call the contract or deploy a new contract. Either of these transactions are capable of calling a large variety of functions. Functions called can be user defined. Data passes in and out of the EVM in either case. This data can pass back to the user or forward on to another contract.
+
+<a href="https://imgur.com/bfKk7mY"><img src="https://i.imgur.com/bfKk7mY.png" title="source: imgur.com" /></a>
+
+The image above shows how a transaction flow can happen in the EVM. Data is passed out of the EVM in what is known as a 'block'. A block is the final result of the arguments, variables, functions, and transactions presented. The data has been manipulated and encrypted. A proof of work puzzle solved by network miners cryptographically proves data integrity. It also cryptographically ties the completed block to adjacent data blocks.
+
+## Miners - The actual computing power of the EVM
+
+Key features of the Ethereum Blockchain include decentralization and peer to peer networking. A node can be run by any individual with hardware meeting minimum system requirements. Transactions are broadcast on to the network so that multiple nodes can see them at the same time. A fee is included for the miner that completes the proof of work and finishes a block. This spider web type of proliferation allows the nodes to compare the information they have received and agree that it matches. This comparison is performed among many nodes to create what is called consensus, or agreement that the values now held in block are the true values that represent the current state of the EVM.
```

Comment:
```suggestion
Key features of the Ethereum Blockchain include decentralization and peer to peer networking. A node can be run by any individual with hardware meeting minimum system requirements. Transactions are broadcast on to the network so that multiple nodes can see them at the same time. A fee is included for the miner that completes the proof of work and finishes a block. This spider web type of proliferation allows the nodes to compare the information they have received and agree that it matches. This comparison is performed among many nodes to create what is called consensus, or agreement that the values now held in block are the true values that represent the current state of the EVM.
```